docs/patterns.rst

```diff
@@ -167,3 +167,12 @@ connecting to a database::
     A new ``setup`` method was added to be able to handle the new
     :class:`~configurations.values.Value` classes and allow an in-between
     modification of the configuration values.
+    
+
+Standalone scripts
+------------------
+
+If you wish to run scripts outside of your project you need to add these lines on the top of your file::
```

Comment:
Thanks - although the wrapping seems to be too hard.. ;)

(Don't you have an editor command for that?
In Vim this can be done with `gqap` to rewrap the paragraph - given correct settings.)